client/Encounter/Encounter.ts

```diff
@@ -208,18 +208,6 @@ export class Encounter {
 
         const nextCombatant = this.Combatants()[nextIndex];
 
-        this.durationTags
-            .filter(t =>
-                t.DurationRemaining() == 0 && (
-                    (t.DurationCombatantId == activeCombatant.Id && t.DurationTiming == EndOfTurn) ||
-                    (t.DurationCombatantId == nextCombatant.Id && t.DurationTiming == StartOfTurn)
-                )
-            )
-            .forEach(t => {
-                t.Remove();
```

Comment:
This isn't quite right. Save shouldn't be side-affecting, it should only strip the tags off of the SavedEncounter that it returns. Since the encounter is saved after every action this winds up immediately trimming off the expired tags.